pkgs/servers/home-assistant/custom-components/climate_group/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildHomeAssistantComponent,
+}:
+buildHomeAssistantComponent rec {
+  owner = "bjrnptrsn";
+  domain = "climate_group";
+  version = "1.0.7";
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = domain;
+    rev = "refs/tags/${version}";
+    hash = "sha256-f/VQUNzRSxmKGNgijaafQ5NbngUUKmcdkafYC3Ol9qM=";
+  };
+
+  dontBuild = true;
+
+  meta = with lib; {
+    changelog = "https://github.com/bjrnptrsn/climate_group/releases/tag/${version}";
+    description = "Home Assistant Climate Group";
```

```suggestion
    description = "Group multiple climate devices to a single entity";
```
seems more descriptive to me